analyzer/src/main/kotlin/managers/NPM.kt

```diff
@@ -159,7 +159,7 @@ class NPM : PackageManager() {
             var homepageUrl: String
             var downloadUrl: String
             var hash: String
-            var hashAlgorithm = ""
+            val hashAlgorithm = ""
```

Comment:
I kept is as a var because it will eventually get a value assigned below, but it's easy to change it back once we get there.